Library/Include/Babylon/Console.h

```diff
@@ -22,25 +26,31 @@ namespace Babylon
             Error,
         };
 
-        static Napi::ObjectReference Create(Napi::Env env, LogHandlerT& handler)
+        using ParentT = Napi::ObjectWrap<Console>;
+        using CallbackT = std::function<void(const char*, LogLevel)>;
+
+        static Napi::ObjectReference AddConsoleToEnv(Napi::Env env, CallbackT callback)
```

Comment:
Perhaps `CreateInstance` for plugins that assigns itself and `CreateConstructor` for plugins that create a type.